Client/Frontend/Settings/CustomSearchViewController.swift

```diff
@@ -118,6 +118,10 @@ class CustomSearchViewController: SettingsTableViewController {
         }
         return nil
     }
+    
+    func updateSaveButton(){
+        self.navigationItem.rightBarButtonItem?.isEnabled = !self.engineTitle.isEmptyOrWhitespace() && !(self.urlString?.isEmptyOrWhitespace() ?? true)
+    }
```

Comment:
Missing space + nit so it's easier to read
```suggestion
    func updateSaveButton() {
        let isEnabled = !self.engineTitle.isEmptyOrWhitespace() && !(self.urlString?.isEmptyOrWhitespace() ?? true)
        self.navigationItem.rightBarButtonItem?.isEnabled = isEnabled
    }
```